infra/charts/turing/values.yaml

```diff
@@ -253,10 +253,6 @@ mlp:
   postgresql:
     # -- Name of MLP's Postgresql deployment
     nameOverride: postgresql-mlp
-    # -- Password for MLP Postgresql database
-    postgresqlPassword: mlp
```

Comment:
This is now moved under a nested `auth` field and is anyway defined directly in the subchart with the same value. I believe it was added here as well as an example, can be cleaned up.